inc/commonitilobject.class.php

```diff
@@ -584,23 +584,23 @@ function countActiveObjectsForSupplier($suppliers_id) {
 
    function cleanDBonPurge() {
 
-      if (!empty($this->grouplinkclass)) {
-         $class = new $this->grouplinkclass();
-         $class->cleanDBonItemDelete($this->getType(), $this->fields['id']);
+      $link_classes = [
+         ITILSolution::class
```

Comment:
Maybe tasks and documents should be handled here. And ITILFollowup (soon in master)

Reply:
Documents relations are handled in CommonDBTM; and it's not possible to handle tasks "proprerly" here (needs some refactoring before).